generators/chipyard/src/main/scala/example/FlatChipTop.scala

```diff
@@ -136,12 +149,17 @@ class FlatChipTop(implicit p: Parameters) extends LazyModule {
       IOCell.generateIOFromSignal(jtag_wire, "jtag", p(IOCellKey), abstractResetAsAsync = true)
     }.get
 
+    ports = ports :+ JTAGPort(() => jtag_pad)
+
     //==========================
     // UART
     //==========================
     require(system.uarts.size == 1)
     val (uart_pad, uartIOCells) = IOCell.generateIOFromSignal(system.module.uart.head, "uart_0", p(IOCellKey))
-
+    val where = PBUS // TODO fix
```

Comment:
Can we fix this before merge?

Reply:
This is actually copied from the existing IOBinder for UART. This comment is because the underlying CanHaveUART code hardcodes the PBUS as the attachment point, instead of parameterizing it. If that were to get fixed, we would fix this too.